var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -75,7 +75,7 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
     depends_on('hiop+mpi', when='+hiop+mpi')
 
     depends_on('petsc@3.13:3.14', when='@:1.2.99')
-    depends_on('petsc@3.16.0:', when='@1.3.0:')
+    depends_on('petsc@3.16.0:3.16.99', when='@1.3.0:')
```

Comment:
```suggestion
    depends_on('petsc@3.16:3.16', when='@1.3.0:')
```
Maybe I'm mis-understanding the syntax here, but I think this is what we want based on the documentation https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges

Reply:
You are right! I didn't know that worked. Thank you.